src/components/structures/IndicatorScrollbar.js

```diff
@@ -152,7 +164,7 @@ export default class IndicatorScrollbar extends React.Component {
                 const additionalScroll = e.deltaY < 0 ? -50 : 50;
 
                 // noinspection JSSuspiciousNameCombination
-                const val = Math.abs(e.deltaY) < 25 && safeToBoost ? (e.deltaY + additionalScroll) : e.deltaY;
+                const val = Math.abs(e.deltaY) < 25 ? (e.deltaY + additionalScroll) : 0;
```

Comment:
this breaks vertical scrolling for non-smoothscrolling browsers (ie: windows)

Reply:
yup, this was a thinko; suggestions welcome

Reply:
instead of zero, `e.deltaY`, like it was before